buildstockbatch/gcp/gcp.py

```diff
@@ -871,6 +959,11 @@ def main():
             help="Only do postprocessing, useful for when the simulations are already done",
             action="store_true",
         )
+        parser.add_argument(
+            "--postprocesscloud",
+            help="If postprocessing is to be done, do it on the cloud",
+            action="store_true",
+        )
```

Comment:
Is there any particular reason for this option to be a command line flag instead of a setting in the project config? That feels to me like the more natural place for it.

Reply:
This is moot since I've changed this to always run on the cloud (as we spoke about offline).